semantic_conventions/metrics/process-runtime-jvm-metrics.yaml

```diff
@@ -156,7 +156,7 @@ groups:
       (unlike `system.cpu.utilization`).
       [Reference](https://docs.oracle.com/en/java/javase/17/docs/api/java.management/java/lang/management/OperatingSystemMXBean.html#getSystemLoadAverage()).
     instrument: gauge
-    unit: "1"
+    unit: "{run_queue_length}"
```

Comment:
sorry, thinking about this more, is `length` a unit?

maybe
* `{queued_runnable_entity}` (taking name from [the javadoc](https://docs.oracle.com/en/java/javase/17/docs/api/java.management/java/lang/management/OperatingSystemMXBean.html#getSystemLoadAverage()))
* `{run_queue_entity}`
* `{run_queue_item}`
* `{run_queue_work}`

Reply:
What about `run_queue_size`? 🤔. Or just `run_queue`?

Reply:
- `run_queue_size` has the same problem as `run_queue_length`
- `run_queue` would not be correct, because we're measuring run queue items, not run queues - even though I like the brevity.

Reply:
Why I suggested `run_queue` is that maybe it's already obvious that it is the "length", "size", "items" in it? That's what a queue holds anyway so maybe it's OK to leave it out. I'm also fine with the current naming.

Reply:
`run_queue_size` or `run_queue` could be metric names, but I'm not sure that they're units? e.g. what does `1 run_queue` mean?